corehq/motech/repeaters/views/repeaters.py

```diff
@@ -64,6 +64,7 @@ def page_context(self):
             'repeaters': self.repeaters,
             'pending_record_count': RepeatRecord.count(self.domain),
             'gefingerpoken': self.request.couch_user.is_superuser or
+                             self.request.couch_user.has_permission(self.domain, 'edit_motech') or
```

Comment:
The template shows the Data Forwarder's "Configure" button if "gefingerpoken" is set.